ui/src/pages/Dashboard/NodeInfo/Node.vue

```diff
@@ -188,16 +277,55 @@ export default {
       return this.pagination.perPage * (this.pagination.currentPage - 1);
     },
     total() {
-      this.paginationTotal(this.tableData.length);
-      return this.tableData.length;
+      let result;
+      if (this.searchQuery !== '') {
+        const temp = [];
+        result = this.fuseSearch.search(`=${this.searchQuery}`);
+        for (let i = 0; i < Object.keys(result).length; i += 1) {
+          temp.push(result[i].item);
+        }
+        result = temp;
+      } else if (this.filters.default === 'node version' && this.filterssign.default === '>=') {
+        result = this.filter1;
+      } else if (this.filters.default === 'node version' && this.filterssign.default === '<') {
+        result = this.filter3;
+      } else if (this.filters.default === 'nodes hashes' && this.filterssign.default === '>=') {
+        result = this.filter2;
+      } else if (this.filters.default === 'nodes hashes' && this.filterssign.default === '<') {
+        result = this.filter4;
+      } else {
+        result = this.tableData;
+      }
+      this.paginationTotal(result.length);
+      return result.length;
     },
   },
   mounted() {
     this.isLoading = true;
     axios
-      .get('https://stats.runonflux.io/fluxinfo?projection=node')
+      .get('https://stats.runonflux.io/fluxinfo?projection=node,flux,appsHashesTotal')
       .then((response) => {
-        this.tableData = response.data.data;
+        this.values = response.data.data;
+      }).then(() => {
+        for (let i = 0; i < this.values.length; i += 1) {
+          try {
+            this.values[i].node.status.network = 'ipv4';
+          } catch (ex) {
```

Comment:
This looks strange. Is it because this.values[i].node.status.network some property may not be defined as of missing data? Can we use .?

Reply:
use .map

Reply:
Yes back then I'm just reading data with values however in clickup there is a ticket that pertains to this one. thanks